docs/mddocs/Quickstart/ollama_quickstart.md

```diff
@@ -80,6 +80,7 @@ You may launch the Ollama service as below:
   export ZES_ENABLE_SYSMAN=1
   source /opt/intel/oneapi/setvars.sh
   export SYCL_CACHE_PERSISTENT=1
+  export LD_LIBRARY_PATH=.:$LD_LIBRARY_PATH

```

Comment:
why we need this ?

Reply:
Looks  not very friendly for users. It would be better if we can hidden this for users.

Reply:
This is to link to the shared library when running `ollama`. I can try to hide this by modifying the compilation method.